Workbooks/AKS/Node Kubelet/settings.json

```diff
@@ -0,0 +1,11 @@
+{
+    "$schema": "https://github.com/Microsoft/Application-Insights-Workbooks/blob/master/schema/settings.json",
+    "name":"Kubelet",
+    "author": "Microsoft",
+    "isPreview": true,
+    "galleries": [
+        { "type": "container-insights", "resourceType": "Microsoft.ContainerService/managedClusters", "order":1100 },
+        { "type": "container-insights", "resourceType": "Microsoft.Kubernetes/connectedClusters", "order": 1100 },
```

Comment:
this is the wrong order number. please check the wiki. increment order # by 100 from the hightest current order number

Reply:
what's the highest order # we have right now? Do I just have to click through and find it?